### PR TITLE
Filter input

### PR DESCRIPTION
**Pull request by AnotherKamila**
[creating a PR so I can merge this]

---
This change is [Reviewable](https://reviewable.io/reviews/anotherkamila/tasksched/30)

### CODE REVIEW

File: src/NextTaskPage.elm [hunk 1]
```diff
@@ -13,6 +13,7 @@ import Taskwarrior.Utils as Taskwarrior
 
 import Html  exposing    (Html, text)
 import Model exposing    (Model, Msg(..))
+import UrlState exposing (toggle_next)
```
Comment (AnotherKamila) [inline src/NextTaskPage.elm]: not needed, correct?
```suggestion
```

Comment (evilham) [reply src/NextTaskPage.elm]: Correct

**Comment by evilham** [comment]
Can we merge this now? :)

File: style.css [hunk 2]
```diff
@@ -42,3 +42,10 @@
     position: relative;
     top: 2px;
 }
+
+/* Ugly hack to align search button.
+ * Without this, the search button is slightly to the top
+ * of the input field and the refresh button. */
+label[for="main-search-expandable"] {
```
Comment (AnotherKamila) [inline style.css:49]: eh, I'd give the thing a class instead of using label, but if you think this is robust enough, sure...

**Pull request merged**